app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java

```diff
@@ -43,21 +44,34 @@
 
     SharedPreferences sharedpreferences;
     SharedPreferences.Editor editor;
+    private Button okButton;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_trustees);
 
-        System.out.println(comradeEditText.size());
-        comradeEditText.add((EditText) findViewById(R.id.comrade1EditText));
```

Comment:
I had initially planned to get hold of the views individually so that I can
request focus on the view next to the currently focused EditText. I found
that there was a reference being made to the EditTexts but they were only
made to add them to the list which is not so useful since I can't make use
of it elsewhere if I'm trying to make any changes to the view in future.
If we think from the optimisation point of view, the newly written code is
more efficient than the earlier written code in terms of reusablility.
On 15-Apr-2016 4:13 pm, "Dinu Kumarasiri" <notifications@github.com> wrote:

> In app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java
> <https://github.com/systers/PCSA-Android/pull/239#discussion_r59857177>:
>
> >
> >      @Override
> >      protected void onCreate(Bundle savedInstanceState) {
> >          super.onCreate(savedInstanceState);
> >          setContentView(R.layout.activity_trustees);
> >
> > -        System.out.println(comradeEditText.size());
> > -        comradeEditText.add((EditText) findViewById(R.id.comrade1EditText));
>
> Explain the rationale behind your choice of removing these and breaking
> the functionality in to two.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/systers/PCSA-Android/pull/239/files/d49225a49514d8dce90e8bb4d5ae7244475cb082#r59857177>
>